apiary.apib

```diff
@@ -4,7 +4,107 @@ FORMAT: 1A
 
 The API for interacting with IPFS nodes.
 
-# Group add
+# Group add [POST /add{?arg}{&recursive,quiet,silent,progress,trickle,only%2Dhash,wrap%2Dwith%2Ddirectory,hidden,chunker,pin}]
```

Comment:
I completely agree with this. Flags are not the best way to send arguments to an HTTP API.